ga4gh/datamodel/datasets.py

```diff
@@ -190,3 +237,69 @@ def _setMetadata(self):
                 except KeyError as err:
                     raise exceptions.MissingDatasetMetadataException(
                         metadataFileName, str(err))
+
+
+class AbstractOntology(object):
+    """
+    The base class of storing an ontology
+    At this moment an "Ontology" is just a map between names and IDs (e.g.
+    in Sequence Ontology we would have "SO:0001583 <-> missense_variant")
+    This is a tempotrary solution adn must be replaced by more comprehensive
+    ontology object.
+    """
+
+    def __init__(self):
+        self._nameIdMap = dict()
+        self._idNameMap = dict()
+
+    def add(self, id_, name):
+        self._nameIdMap[id_] = name
+        self._idNameMap[name] = id_
+
+    def getId(self, name):
+        return self._idNameMap[name]
+
+    def getName(self, id_):
+        return self._nameIdMap[id_]
+
+
+class FileSystemOntology(object):
```

Comment:
I'm still a bit confused by this `Ontology` hierarchy here, and I think we might be able to simplify it a bit. As I understand it, currently an Ontology is a one-to-one mapping between OTs and names and corresponds to one text file, e.g, `sequence_ontology.txt`. Why don't we just have ``datadir/ontologies/sequence_ontology.txt`` instead of having the intermediate directory?

Also, I'm confused by the distinction between the FileSystem ontology (which seems to be a mapping from OntologyName -> Ontology objects) and the Ontology itself. I think the FileSystemOntology object should map directly to the file full of terms, and we remove the AbstractOntology class (which isn't really abstract, as we instantiate it directly).

I'll comment a few other places where I think we can clarify this.